### PR TITLE
Improved the output file names

### PR DESCRIPTION
**Pull request by micheles**
As requested by the risk team, https://github.com/gem/oq-risklib/issues/464. Notice that I am changing only the names of the files generated by the oq-lite calculators. The old calculators continue to use the old names they always had. The calculation ID has been added to the exported file name as a suffix.

The demos are running here: https://ci.openquake.org/job/zdevel_oq-risklib/1151/console

### CODE REVIEW

**Comment by ptormene** [comment]
Please check about the conflicts to be resolved

File: openquake/commonlib/export/hazard.py [hunk 1]
```diff
@@ -464,22 +461,21 @@ def export_gmf(ekey, dstore):
...
                              imtls, investigation_time):
     """
     Export the curves of the given realization into XML.
 
     :param key: output_type and export_type
-    :param export_dir: the directory where to export
-    :param fname: name of the exported file
+    :param dest: name of the exported file
     :param sitecol: site collection
     :param rlz: realization instance
```
Comment (ptormene) [inline openquake/commonlib/export/hazard.py]: Please remove the parameter `rlz` from the docstring

File: openquake/commonlib/export/hazard.py [hunk 2]
```diff
@@ -301,7 +297,7 @@ def export_gmf_csv(key, export_dir, fname, sitecol, ruptures, gmfs, rlz,
 HazardCurve = collections.namedtuple('HazardCurve', 'location poes')
 
 
-def export_hazard_curves_csv(key, export_dir, fname, sitecol, curves_by_imt,
+def export_hazard_curves_csv(key, dest, sitecol, curves_by_imt,
```
Comment (ptormene) [inline openquake/commonlib/export/hazard.py]: Please update the docstring accordingly

File: openquake/commonlib/datastore.py [hunk 3]
```diff
@@ -213,14 +213,19 @@ def create_dset(self, key, dtype, size=None):
...
-    def export_path(self, key, fmt):
+    def export_path(self, name, export_dir=None):
         """
-        Return the name of the exported file.
+        Return the path of the exported file by adding the export_dir in
+        front, the calculation ID at the end.
 
-        :param key: the datastore key
-        :param fmt: the export format extension
+        :param fname: file name template
```
Comment (ptormene) [inline openquake/commonlib/datastore.py]: Please fix the inconsistency between `name` and `fname`

**Comment by ptormene** [comment]
Apart from the minor fixes to the docstrings, if tests are green, LGTM

File: openquake/commonlib/export/hazard.py (hunk 2 above)
Comment (micheles) [inline openquake/commonlib/export/hazard.py]: Done

**Pull request merged**